File: app/src/main/java/com/example/kineduexample/data/network/model/RelatedActivities.java
package com.example.kineduexample.data.network.model;

import com.google.gson.annotations.SerializedName;

class RelatedActivities {
    @SerializedName("id")
    private Integer id;

    @SerializedName("name")
    private String name;

    @SerializedName("description")
    private String description;

    @SerializedName("purpose")
    private String purpose;

    @SerializedName("age")
    private Integer age;

    @SerializedName("age_group")
    private String ageGroup;

    @SerializedName("activity_type")
    private String activityType;

    @SerializedName("video_provider_thumbnails")
    private VideoProviderThumbnails videoProviderThumbnails;

    @SerializedName("video_id")
    private String videoId;

    @SerializedName("area_id")
    private Integer areaId;

    @SerializedName("shareable_video_url")
    private String shareableVideoUrl;

    @SerializedName("locked")
    private Boolean locked;

    @SerializedName("picture")
    private String picture;

    @SerializedName("faved")
    private Boolean faved;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getPurpose() {
        return purpose;
    }

    public void setPurpose(String purpose) {
        this.purpose = purpose;
    }

    public Integer getAge() {
        return age;
    }

    public void setAge(Integer age) {
        this.age = age;
    }

    public String getAgeGroup() {
        return ageGroup;
    }

    public void setAgeGroup(String ageGroup) {
        this.ageGroup = ageGroup;
    }

    public String getActivityType() {
        return activityType;
    }

    public void setActivityType(String activityType) {
        this.activityType = activityType;
    }

    public VideoProviderThumbnails getVideoProviderThumbnails() {
        return videoProviderThumbnails;
    }

    public void setVideoProviderThumbnails(VideoProviderThumbnails videoProviderThumbnails) {
        this.videoProviderThumbnails = videoProviderThumbnails;
    }

    public String getVideoId() {
        return videoId;
    }

    public void setVideoId(String videoId) {
        this.videoId = videoId;
    }

    public Integer getAreaId() {
        return areaId;
    }

    public void setAreaId(Integer areaId) {
        this.areaId = areaId;
    }

    public String getShareableVideoUrl() {
        return shareableVideoUrl;
    }

    public void setShareableVideoUrl(String shareableVideoUrl) {
        this.shareableVideoUrl = shareableVideoUrl;
    }

    public Boolean getLocked() {
        return locked;
    }

    public void setLocked(Boolean locked) {
        this.locked = locked;
    }

    public String getPicture() {
        return picture;
    }

    public void setPicture(String picture) {
        this.picture = picture;
    }

    public Boolean getFaved() {
        return faved;
    }

    public void setFaved(Boolean faved) {
        this.faved = faved;
    }

    @Override
    public String toString() {
        return "RelatedActivities{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", description='" + description + '\'' +
                ", purpose='" + purpose + '\'' +
                ", age=" + age +
                ", ageGroup='" + ageGroup + '\'' +
                ", activityType='" + activityType + '\'' +
                ", videoProviderThumbnails=" + videoProviderThumbnails +
                ", videoId='" + videoId + '\'' +
                ", areaId=" + areaId +
                ", shareableVideoUrl='" + shareableVideoUrl + '\'' +
                ", locked=" + locked +
                ", picture='" + picture + '\'' +
                ", faved=" + faved +
                '}';
    }
}

File: app/src/main/java/com/example/kineduexample/data/network/model/RelatedActivities.kt
package com.example.kineduexample.data.network.model

import com.google.gson.annotations.SerializedName

class RelatedActivities {
    @SerializedName("id")
    var id: Int? = null

    @SerializedName("name")
    var name: String? = null

    @SerializedName("description")
    var description: String? = null

    @SerializedName("purpose")
    var purpose: String? = null

    @SerializedName("age")
    var age: Int? = null

    @SerializedName("age_group")
    var ageGroup: String? = null

    @SerializedName("activity_type")
    var activityType: String? = null

    @SerializedName("video_provider_thumbnails")
    var videoProviderThumbnails: VideoProviderThumbnails? = null

    @SerializedName("video_id")
    var videoId: String? = null

    @SerializedName("area_id")
    var areaId: Int? = null

    @SerializedName("shareable_video_url")
    var shareableVideoUrl: String? = null

    @SerializedName("locked")
    var locked: Boolean? = null

    @SerializedName("picture")
    var picture: String? = null

    @SerializedName("faved")
    var faved: Boolean? = null

    override fun toString(): String {
        return "RelatedActivities{" +
                "id=" + id +
                ", name='" + name + '\''.toString() +
                ", description='" + description + '\''.toString() +
                ", purpose='" + purpose + '\''.toString() +
                ", age=" + age +
                ", ageGroup='" + ageGroup + '\''.toString() +
                ", activityType='" + activityType + '\''.toString() +
                ", videoProviderThumbnails=" + videoProviderThumbnails +
                ", videoId='" + videoId + '\''.toString() +
                ", areaId=" + areaId +
                ", shareableVideoUrl='" + shareableVideoUrl + '\''.toString() +
                ", locked=" + locked +
                ", picture='" + picture + '\''.toString() +
                ", faved=" + faved +
                '}'.toString()
    }
}
